lib/remap-functions/src/lib.rs

```diff
@@ -350,6 +354,10 @@ pub fn all() -> Vec<Box<dyn remap::Function>> {
         Box::new(Merge),
         #[cfg(feature = "now")]
         Box::new(Now),
+        // We are not sure if this is the way we want to expose this functionality yet
```

Comment:
I added the references to `only_fields` here with this comment just to avoid confusion about where or not it should be exported. I was going to write a bench for it, hence the exporting above, but decided against it since it is unclear if it will be sticking around.